proposals/0240-sdl-js-pwa.md

```diff
@@ -345,12 +345,12 @@ Many services are available over a web application and modern WebEngines provide
   </param>
 </function>
 </interface>
-```
+```f
```

Comment:
I think this is a typo.  Please fix.

Reply:
Sorry for this typo I was doing ctrl+f too much...